framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -249,6 +249,15 @@
      */
     private final Map<String, AnnotationMirror> aliases = new HashMap<String, AnnotationMirror>();
 
+    /** Set contains class names of aliases whose elements need to be copied over. */
+    private final Set<String> aliasesCopyElements = new HashSet<>();
+
+    /**
+     * Map from class name (canonical name) of an annotation, to the ignorable elements that the
```

Comment:
Shouldn't the `canonical name` be `alias name`? Or to make it more symmetric to the other documentation: `Map from class names of aliases of an annotation to the ignorable...`?

Reply:
same reason here as above